src/Config.php

```diff
@@ -3661,7 +3666,7 @@ public function showFormManagement()
             [
                 'value' => $CFG_GLPI['document_max_size'],
                 'min'   => 1,
-                'max'   => 250,
+                'max'   => $upload_max_filesize,
```

Comment:
Purpose of #8180 was to be able to upload files larger than `upload_max_filesize`. Indeed, files are send in multiple chunks, so you can probably remove this limit safely.